src/pages/_layout-dev/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import classNames from 'classnames'
+import SidebarSidecarLayout from 'layouts/docs/partials/sidebar-sidecar-layout'
+import s from './style.module.css'
+
+function DemoContent({ name }: { name: string }): React.ReactElement {
+  const [isLong, setIsLong] = useState(false)
+  return (
+    <div className={classNames(s[`${name}Demo`], { [s.isLong]: isLong })}>
+      {name} Content
+      <br />
+      <button type="button" onClick={() => setIsLong(!isLong)}>
+        Demo {isLong ? 'short' : 'long'} {name} Content
+      </button>
+    </div>
+  )
+}
+
+function DemoLayoutDev(): React.ReactElement {
```

Comment:
Intention is to delete this, started this as a skateboard version of getting the layout working the way I think we want it to. Leaving it up for now to make it easier to isolate and debug layout arrangement concerns from docs page things.